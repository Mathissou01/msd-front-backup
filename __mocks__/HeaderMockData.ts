import { GetContractMenuDocument } from "../src/graphql/codegen/generated-types";

export const defaultMockData = [
  {
    request: {
      query: GetContractMenuDocument,
      variables: {
        contractId: "1",
      },
    },
    result: {
      data: {
        contract: {
          data: {
            attributes: {
              contractMenu: {
                data: {
                  attributes: {
                    serviceLinks: [
                      {
                        name: "Carte",
                        isDisplayed: true,
                        picto: {
                          data: {
                            attributes: {
                              url: "https://stomsdmediadev.blob.core.windows.net/msd-media-dev/assets-local-grymas/address.svg",
                            },
                          },
                        },
                      },
                      {
                        name: "Collecte à mon adresse",
                        isDisplayed: true,
                        picto: {
                          data: null,
                        },
                      },
                      {
                        name: "Guide du tri",
                        isDisplayed: true,
                        picto: {
                          data: null,
                        },
                      },
                      {
                        name: "Faire une demande",
                        isDisplayed: true,
                        picto: {
                          data: null,
                        },
                      },
                      {
                        name: "Réduire mes déchets",
                        isDisplayed: true,
                        picto: {
                          data: null,
                        },
                        freeContentSubService: {
                          data: {
                            id: "2",
                            attributes: {
                              name: "Réduire mes déchets",
                            },
                          },
                        },
                      },
                      {
                        name: "Valoriser mes déchets",
                        isDisplayed: true,
                        picto: {
                          data: null,
                        },
                        freeContentSubService: {
                          data: {
                            id: "1",
                            attributes: {
                              name: "Valoriser mes déchets",
                            },
                          },
                        },
                      },
                    ],
                  },
                },
              },
            },
          },
        },
      },
    },
  },
];
