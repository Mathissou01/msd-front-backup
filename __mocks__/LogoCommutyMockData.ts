import { GetLogoDocument } from "../src/graphql/codegen/generated-types";

export const defaultMockData = [
  {
    request: {
      query: GetLogoDocument,
      variables: {
        contractId: "1",
      },
    },
    result: {
      data: {
        contract: {
          data: {
            id: "1",
            attributes: {
              logo: {
                data: {
                  id: "24",
                  attributes: {
                    name: "Votre-collectivite_large.png",
                    mime: "image/png",
                    size: 21.91,
                    url: "https://stomsdmediadev.blob.core.windows.net/msd-media-dev/assets-local-sdial/Votre-collectivite_large.png",
                    provider: "azure-storage-blob",
                    hash: "Votre_collectivite_large_40e95434cd",
                    alternativeText: "Votre-collectivite_large.png",
                  },
                },
              },
            },
          },
        },
      },
    },
  },
];
