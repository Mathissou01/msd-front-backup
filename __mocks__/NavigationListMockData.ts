import { GetContractMenuDocument } from "../src/graphql/codegen/generated-types";

export const defaultMockData = [
  {
    request: {
      query: GetContractMenuDocument,
      variables: {
        contractId: "1",
      },
    },
    result: {
      data: {
        contract: {
          __typename: "ContractEntityResponse",
          data: {
            __typename: "ContractEntity",
            attributes: {
              __typename: "Contract",
              contractMenu: {
                __typename: "ContractMenuEntityResponse",
                data: {
                  __typename: "ContractMenuEntity",
                  attributes: {
                    __typename: "ContractMenu",
                    serviceLinks: [
                      {
                        __typename: "ComponentLinksMap",
                        name: "Carte",
                        isDisplayed: true,
                        picto: {
                          __typename: "UploadFileEntityResponse",
                          data: {
                            __typename: "UploadFileEntity",
                            attributes: {
                              __typename: "UploadFile",
                              url: "https://fake-url/address.svg",
                            },
                          },
                        },
                      },
                      {
                        __typename: "ComponentLinksCalendar",
                        name: "Collecte à mon adresse",
                        isDisplayed: true,
                        picto: {
                          __typename: "UploadFileEntityResponse",
                          data: null,
                        },
                      },
                      {
                        __typename: "ComponentLinksRecycling",
                        name: "Guide du tri",
                        isDisplayed: true,
                        picto: {
                          __typename: "UploadFileEntityResponse",
                          data: null,
                        },
                      },
                      {
                        __typename: "ComponentLinksRequest",
                        name: "Faire une demande",
                        isDisplayed: true,
                        picto: {
                          __typename: "UploadFileEntityResponse",
                          data: null,
                        },
                      },
                      {
                        __typename: "ComponentLinksFrees",
                        name: "Réduire mes déchets",
                        isDisplayed: true,
                        picto: {
                          __typename: "UploadFileEntityResponse",
                          data: null,
                        },
                        freeContentSubService: {
                          __typename: "FreeContentSubServiceEntityResponse",
                          data: {
                            __typename: "FreeContentSubServiceEntity",
                            id: "2",
                            attributes: {
                              __typename: "FreeContentSubService",
                              name: "Réduire mes déchets",
                            },
                          },
                        },
                      },
                      {
                        __typename: "ComponentLinksFrees",
                        name: "Valoriser mes déchets",
                        isDisplayed: true,
                        picto: {
                          __typename: "UploadFileEntityResponse",
                          data: null,
                        },
                        freeContentSubService: {
                          __typename: "FreeContentSubServiceEntityResponse",
                          data: {
                            __typename: "FreeContentSubServiceEntity",
                            id: "1",
                            attributes: {
                              __typename: "FreeContentSubService",
                              name: "Valoriser mes déchets",
                            },
                          },
                        },
                      },
                    ],
                  },
                },
              },
            },
          },
        },
      },
    },
  },
];
