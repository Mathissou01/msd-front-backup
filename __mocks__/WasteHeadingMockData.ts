import {
  Flow,
  UploadFileEntityResponse,
} from "../src/graphql/codegen/generated-types";

export const defaultMockData = {
  title: "Edito heading title",
  picto: {
    data: {
      attributes: {
        url: "/images/images-temp/temp_image.jpg",
        alternativeText: "",
        hash: "",
        mime: "",
        name: "",
        provider: "",
        size: 0,
      },
    },
  } as UploadFileEntityResponse,
  collecItems: {
    __typename: "Flow",
    name: "Déchets d’équipements électriques et électroniques",
    recyclingGesture: "no_trash",
    collectVoluntaries: {
      data: [
        {
          __typename: "CollectVoluntaryEntity",
          id: "1",
          attributes: {
            __typename: "CollectVoluntary",
            name: "Colonne",
            picto: {
              __typename: "UploadFileEntityResponse",
              data: {
                __typename: "UploadFileEntity",
                id: "17",
                attributes: {
                  __typename: "UploadFile",
                  name: "decheterie_496861.svg",
                  hash: "decheterie_496861_8571ce080c",
                  mime: "image/svg+xml",
                  size: 2.38,
                  url: "https://stomsdmediadev.blob.core.windows.net/msd-media-dev/assets-local-sdial/decheterie_496861.svg",
                  provider: "azure-storage-blob",
                  alternativeText: null,
                },
              },
            },
          },
        },
      ],
    },
    collectDropOffs: {
      data: [
        {
          id: "2",
          attributes: {
            name: "Déchèterie",
            picto: {
              data: [
                {
                  id: "17",
                  attributes: {
                    url: "/images/images-temp/temp_image.jpg",
                    alternativeText: "",
                    hash: "",
                    mime: "",
                    name: "",
                    provider: "",
                    size: 0,
                  },
                },
              ],
            },
          },
        },
      ],
    },
    collectDoorToDoors: {
      __typename: "CollectDoorToDoorRelationResponseCollection",
      data: [
        {
          __typename: "CollectDoorToDoorEntity",
          id: "1",
          attributes: {
            __typename: "CollectDoorToDoor",
            name: "Sac",
            picto: {
              __typename: "UploadFileEntityResponse",
              data: {
                __typename: "UploadFileEntity",
                id: "17",
                attributes: {
                  __typename: "UploadFile",
                  name: "decheterie_496861.svg",
                  hash: "decheterie_496861_8571ce080c",
                  mime: "image/svg+xml",
                  size: 2.38,
                  url: "https://stomsdmediadev.blob.core.windows.net/msd-media-dev/assets-local-sdial/decheterie_496861.svg",
                  provider: "azure-storage-blob",
                  alternativeText: null,
                },
              },
            },
          },
        },
      ],
    },
  } as Flow,
};
