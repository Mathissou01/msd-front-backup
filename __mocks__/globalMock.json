{
  "contract": {
    "id": "1",
    "attributes": {
      "clientName": "Grand Test",
      "clientType": "city",
      "contractStatus": "Initialisation",
      "isNonExclusive": false,
      "isRVFrance": true,
      "logo": {
        "data": {
          "id": "26",
          "attributes": {
            "name": "images.png",
            "mime": "image/png",
            "size": 2.24,
            "url": "https://stomsdmediadev.blob.core.windows.net/msd-media-dev/assets-local-root/images.png",
            "provider": "azure-storage-blob",
            "hash": "images_eb8049ec37",
            "alternativeText": null
          }
        }
      },
      "contractCustomization": {
        "data": {
          "id": "2",
          "attributes": {
            "footer": {
              "data": {
                "attributes": {
                  "accessibilityLevel": null,
                  "cguSubService": {
                    "data": {
                      "id": "1",
                      "attributes": {
                        "name": "CGU",
                        "link": null
                      }
                    }
                  },
                  "accessibilitySubService": {
                    "data": {
                      "id": "1",
                      "attributes": {
                        "name": "Accessibilité",
                        "link": null
                      }
                    }
                  },
                  "confidentialitySubService": {
                    "data": {
                      "id": "1",
                      "attributes": {
                        "name": "Confidentialité",
                        "link": null
                      }
                    }
                  },
                  "cookiesSubService": {
                    "data": {
                      "id": "1",
                      "attributes": {
                        "name": "Cookies",
                        "link": null
                      }
                    }
                  },
                  "contactUsSubService": {
                    "data": {
                      "id": "1",
                      "attributes": {
                        "name": "Contactez-nous",
                        "label": "Contactez-nous",
                        "link": null,
                        "isActivated": false
                      }
                    }
                  }
                }
              }
            }
          }
        }
      },
      "contractMenu": {
        "data": {
          "attributes": {
            "serviceLinks": [
              {
                "__typename": "ComponentLinksRecyclingGuide",
                "id": "3",
                "name": "Guide du tri",
                "isDisplayed": true,
                "picto": {
                  "data": null
                }
              },
              {
                "__typename": "ComponentLinksNews",
                "id": "2",
                "name": "Actualités",
                "isDisplayed": true,
                "picto": {
                  "data": null
                }
              },
              {
                "__typename": "ComponentLinksEvents",
                "id": "2",
                "name": "Évenements",
                "isDisplayed": true,
                "picto": {
                  "data": null
                }
              },
              {
                "__typename": "ComponentLinksFrees",
                "id": "3",
                "name": "Editorial 1",
                "isDisplayed": true,
                "freeContents": {
                  "data": [
                    {
                      "id": "1",
                      "attributes": {
                        "name": "Editorial 1"
                      }
                    }
                  ]
                },
                "picto": {
                  "data": null
                }
              },
              {
                "__typename": "ComponentLinksFrees",
                "id": "4",
                "name": "Editorial 2",
                "isDisplayed": true,
                "freeContents": {
                  "data": [
                    {
                      "id": "2",
                      "attributes": {
                        "name": "Editorial 2"
                      }
                    }
                  ]
                },
                "picto": {
                  "data": null
                }
              }
            ]
          }
        }
      }
    }
  },
  "colors": {
    "external-contrast-text": "#FFFFFF",
    "external-primary": "#0A959F",
    "calculated-primary-dark": "#006771",
    "calculated-primary-light": "#0a959f1a",
    "external-secondary": "#A3C037",
    "calculated-secondary-dark": "#719000",
    "calculated-secondary-light": "#a3c0371a",
    "external-waste-household": "#6C757D",
    "external-waste-selective": "#FFC229",
    "external-waste-bio": "#BD4602",
    "external-waste-green": "#9BCD41",
    "external-waste-glass": "#30A23D",
    "external-waste-paper": "#2068C7",
    "external-waste-other": "#FFFFFF"
  }
}
