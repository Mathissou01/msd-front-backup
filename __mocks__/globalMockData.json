{
  "contractMenu": {
    "data": {
      "attributes": {
        "serviceLinks": [
          {
            "__typename": "ComponentLinksMap",
            "name": "Carte",
            "isDisplayed": true,
            "picto": {
              "data": null
            }
          },
          {
            "__typename": "ComponentLinksCalendar",
            "name": "Collecte à mon adresse",
            "isDisplayed": true,
            "picto": {
              "data": null
            }
          },
          {
            "__typename": "ComponentLinksRequest",
            "name": "Faire une demande",
            "isDisplayed": true,
            "picto": {
              "data": null
            }
          },
          {
            "__typename": "ComponentLinksRecycling",
            "name": "Guide du tri",
            "isDisplayed": true,
            "picto": {
              "data": null
            }
          },
          {
            "__typename": "ComponentLinksFrees",
            "name": "Réduire mes déchets",
            "isDisplayed": true,
            "picto": {
              "data": null
            }
          },
          {
            "__typename": "ComponentLinksFrees",
            "name": "Valoriser mes déchets",
            "isDisplayed": true,
            "picto": {
              "data": null
            }
          }
        ]
      }
    }
  }
}
