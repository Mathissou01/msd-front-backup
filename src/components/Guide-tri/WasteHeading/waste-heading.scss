@use "sass:list";
@use "sass:math";
@use "src/styles/01-settings/settings.colors" as colors;
@use "src/styles/01-settings/settings.blobs" as b;
@use "src/styles/02-tools/tools.shadows" as shadows;
@use "src/styles/02-tools/tools.include-media" as im;
@use "src/styles/02-tools/tools.z-index" as index;
@use "src/styles/02-tools/tools.blobs" as blobs;

.c-WasteHeading {
  $random-start: math.random(6);

  position: relative;
  padding: 40px 16px;
  background-color: colors.$background-dark;

  @for $i from 1 through (list.length(b.$blobs) * 2) {
    *:nth-child(#{(length(b.$blobs) * 2)}n + #{$i}) {
      // stylelint-disable-next-line
      .o-Blob::after {
        @include blobs.blob((($random-start + $i) % list.length(b.$blobs)) + 1);

        transform: rotate(#{math.random(360deg)}deg);
      }
    }
  }

  @include im.media(">=tablet") {
    padding: 80px 124px;
  }

  &__DetailDechet {
    @include shadows.shadow(1);

    position: relative;
    z-index: index.z("site-content");
    display: flex;
    flex-direction: column;
    gap: 24px;
    justify-content: center;
    padding: 24px 16px 100px;
    overflow: hidden;
    background-color: colors.$white;
    border-radius: 8px;

    @include im.media(">=tablet") {
      gap: 36px;
      padding: 36px 24px 24px;
    }
  }

  &__Content {
    width: 100%;

    @include im.media(">=tablet") {
      display: flex;
      flex-direction: row;
      gap: 24px;
      align-items: center;
    }
  }

  &__BlocConsignes {
    display: flex;
    flex-direction: column;

    @include im.media(">=tablet") {
      flex-direction: row;
    }
  }

  &__Bloc {
    display: flex;
    flex-direction: column;
    gap: 16px;
    justify-content: center;
    padding: 16px 24px;
    border: 1px solid colors.$gray-200;

    @include im.media(">=tablet") {
      flex-direction: row;
      align-items: center;
    }

    &_left {
      gap: 16px;
      border-radius: 8px 0 0 8px;
    }

    &_right {
      flex-direction: column;
      gap: 8px;
      align-items: flex-start;
      border-radius: 0 8px 8px 0;
    }
  }

  &__GesturePicto {
    display: flex;
    gap: 16px;
  }

  &__WastePicto {
    position: absolute;
    right: -2px;
    bottom: -2px;
    display: flex;
    align-items: center;
    justify-content: center;
    width: 76px;
    height: 76px;
    margin: 17px;

    @include im.media(">=tablet") {
      top: 0;
      right: 0;
      width: 130px;
      height: 130px;
    }

    &::after {
      // Position the o-Blob::after element
      top: -42px;
      left: -45px;
      z-index: index.z("site-content");
      background-color: colors.$green;

      @include im.media(">=tablet") {
        width: 224px;
        height: 224px;
      }
    }

    &_mobile {
      display: inline;

      @include im.media(">=tablet") {
        display: none;
      }
    }

    &_desktop {
      position: relative;
      display: none;

      @include im.media(">=tablet") {
        display: initial;
      }
    }
  }

  &__BoxContainers,
  &__BoxPictos {
    display: flex;
    flex-wrap: wrap;
    gap: 8px;
    align-items: center;
  }

  &__Picto {
    padding: 5px;
    border: 1px solid colors.$gray-600;
    border-radius: 8px;
  }

  &__Description {
    display: flex;
    flex-wrap: wrap;
    gap: 5px;
  }

  &__NamePicto {
    white-space: nowrap;
  }

  &__SvgContainer {
    position: absolute;
    top: 0;
    left: 0;
    height: 100%;
    pointer-events: none;

    @include im.media(">=tablet") {
      width: unset;
    }

    svg {
      width: 100%;

      #primary-color {
        fill: colors.$brand-primary;
      }

      #secondary-color {
        fill: colors.$brand-secondary;
      }
    }
  }

  &__Svg {
    &_mobile {
      display: inline;

      @include im.media(">=tablet") {
        display: none;
      }
    }

    &_desktop {
      position: relative;
      display: none;

      @include im.media(">=tablet") {
        display: initial;
      }
    }
  }
}
