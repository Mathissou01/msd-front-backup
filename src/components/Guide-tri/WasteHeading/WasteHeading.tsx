import React from "react";
import Image from "next/image";
import {
  Flow,
  Maybe,
  UploadFileEntityResponse,
} from "../../../graphql/codegen/generated-types";
import { makePublicAssetPath } from "../../../lib/utilities";
import { filterAndDisplayPictos } from "../../../lib/waste-pictos";
import MobileDechetHero from "public/images/mobile_dechethero.svg";
import DesktopDechetHero from "public/images/desktop_dechethero.svg";
import "./waste-heading.scss";

interface IWasteHeadingProps {
  title: string;
  gestureText?: Maybe<string> | undefined;
  picto?: Maybe<UploadFileEntityResponse>;
  collectItems?: Maybe<Flow> | undefined;
}
export default function WasteHeading({
  title,
  gestureText,
  picto,
  collectItems,
}: IWasteHeadingProps) {
  /* Static DATA */
  const filteredFlowData = filterAndDisplayPictos(collectItems);

  return (
    <div className="c-WasteHeading">
      <div className="c-WasteHeading__SvgContainer">
        <DesktopDechetHero className="c-WasteHeading__Svg_desktop" />
        <MobileDechetHero className="c-WasteHeading__Svg_mobile" />
      </div>
      <div className="c-WasteHeading__DetailDechet">
        <h1 className="c-WasteHeading__WasteName">{title}</h1>
        <div className="c-WasteHeading__Content">
          <div className="c-WasteHeading__BlocConsignes">
            {gestureText && (
              <div className="c-WasteHeading__Bloc c-WasteHeading__Bloc_left">
                {/*TODO Gesture picto not defined */}
                {/* <div className="c-WasteHeading__GesturePicto">
                  <Image
                    src={"/images/pictos-temp/poubelle-barree.svg"}
                    alt=""
                    width="48"
                    height="48"
                  />
                </div> */}
                <span className="WasteHeading__GestureText">{gestureText}</span>
              </div>
            )}
            <div className="c-WasteHeading__Bloc c-WasteHeading__Bloc_right">
              <div className="c-WasteHeading__BoxContainers">
                {filteredFlowData &&
                  filteredFlowData.pictos.map((picto, index) => {
                    if (picto.attributes?.url) {
                      return (
                        <div key={index} className="c-WasteHeading__BoxPictos">
                          <Image
                            src={makePublicAssetPath(picto.attributes.url)}
                            alt={picto.attributes.alternativeText ?? ""}
                            width="48"
                            height="48"
                            className="c-WasteHeading__Picto"
                          />
                          {index !== filteredFlowData.pictos.length - 1 && (
                            <span>ou</span>
                          )}
                        </div>
                      );
                    }
                  })}
              </div>
              <div className="c-WasteHeading__Description">
                {filteredFlowData &&
                  filteredFlowData.names.map((name, index) => (
                    <div className="c-WasteHeading__Description" key={index}>
                      <span className="c-WasteHeading__NamePicto">{name}</span>
                      {index !== filteredFlowData.names.length - 1 && (
                        <span>ou</span>
                      )}
                    </div>
                  ))}
              </div>
            </div>
          </div>
        </div>

        <div className="o-Blob c-WasteHeading__WastePicto">
          {picto?.data?.attributes?.url && (
            <>
              <Image
                src={makePublicAssetPath(picto?.data?.attributes?.url)}
                alt={picto.data.attributes.alternativeText ?? ""}
                width="130"
                height="130"
                className="c-WasteHeading__WastePicto_desktop"
              />
              <Image
                src={makePublicAssetPath(picto?.data?.attributes?.url)}
                alt={picto.data.attributes.alternativeText ?? ""}
                width="76"
                height="76"
                className="c-WasteHeading__WastePicto_mobile"
              />
            </>
          )}
        </div>
      </div>
    </div>
  );
}
