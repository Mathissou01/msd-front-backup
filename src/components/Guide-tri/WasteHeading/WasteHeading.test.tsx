import { render } from "@testing-library/react";
import { defaultMockData } from "../../../../__mocks__/WasteHeadingMockData";
import WasteHeading from "./WasteHeading";

describe("WasteHeading", () => {
  it("renders", () => {
    const { container } = render(
      <WasteHeading
        title={defaultMockData.title}
        picto={defaultMockData.picto}
        collectItems={defaultMockData.collecItems}
      />,
    );
    expect(container).toMatchSnapshot();
  });
});
