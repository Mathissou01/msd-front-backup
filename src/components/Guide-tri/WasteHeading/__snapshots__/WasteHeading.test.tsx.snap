// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WasteHeading renders 1`] = `
<div>
  <div
    class="c-WasteHeading"
  >
    <div
      class="c-WasteHeading__SvgContainer"
    >
      <div
        class="c-WasteHeading__Svg_desktop"
      />
      <div
        class="c-WasteHeading__Svg_mobile"
      />
    </div>
    <div
      class="c-WasteHeading__DetailDechet"
    >
      <h1
        class="c-WasteHeading__WasteName"
      >
        Edito heading title
      </h1>
      <div
        class="c-WasteHeading__Content"
      >
        <div
          class="c-WasteHeading__BlocConsignes"
        >
          <div
            class="c-WasteHeading__Bloc c-WasteHeading__Bloc_right"
          >
            <div
              class="c-WasteHeading__BoxContainers"
            >
              <div
                class="c-WasteHeading__BoxPictos"
              >
                <img
                  alt=""
                  class="c-WasteHeading__Picto"
                  data-nimg="1"
                  decoding="async"
                  height="48"
                  loading="lazy"
                  src="https://stomsdmediadev.blob.core.windows.net/msd-media-dev/assets-local-sdial/decheterie_496861.svg"
                  style="color: transparent;"
                  width="48"
                />
                <span>
                  ou
                </span>
              </div>
              <div
                class="c-WasteHeading__BoxPictos"
              >
                <img
                  alt=""
                  class="c-WasteHeading__Picto"
                  data-nimg="1"
                  decoding="async"
                  height="48"
                  loading="lazy"
                  src="https://stomsdmediadev.blob.core.windows.net/msd-media-dev/assets-local-sdial/decheterie_496861.svg"
                  style="color: transparent;"
                  width="48"
                />
              </div>
            </div>
            <div
              class="c-WasteHeading__Description"
            >
              <div
                class="c-WasteHeading__Description"
              >
                <span
                  class="c-WasteHeading__NamePicto"
                >
                  Sac
                </span>
                <span>
                  ou
                </span>
              </div>
              <div
                class="c-WasteHeading__Description"
              >
                <span
                  class="c-WasteHeading__NamePicto"
                >
                  Colonne
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="o-Blob c-WasteHeading__WastePicto"
      >
        <img
          alt=""
          class="c-WasteHeading__WastePicto_desktop"
          data-nimg="1"
          decoding="async"
          height="130"
          loading="lazy"
          src="/_next/image?url=%2Fimages%2Fimages-temp%2Ftemp_image.jpg&w=384&q=75"
          srcset="/_next/image?url=%2Fimages%2Fimages-temp%2Ftemp_image.jpg&w=256&q=75 1x, /_next/image?url=%2Fimages%2Fimages-temp%2Ftemp_image.jpg&w=384&q=75 2x"
          style="color: transparent;"
          width="130"
        />
        <img
          alt=""
          class="c-WasteHeading__WastePicto_mobile"
          data-nimg="1"
          decoding="async"
          height="76"
          loading="lazy"
          src="/_next/image?url=%2Fimages%2Fimages-temp%2Ftemp_image.jpg&w=256&q=75"
          srcset="/_next/image?url=%2Fimages%2Fimages-temp%2Ftemp_image.jpg&w=96&q=75 1x, /_next/image?url=%2Fimages%2Fimages-temp%2Ftemp_image.jpg&w=256&q=75 2x"
          style="color: transparent;"
          width="76"
        />
      </div>
    </div>
  </div>
</div>
`;
